kapprover/kapprover.go

```diff
@@ -105,6 +158,7 @@ func tryApprove(filters inspectors.Inspectors, deniers inspectors.Inspectors, wa
 				message, _ := warner.Inspector.Inspect(client, request)
 				if message != "" {
 					log.Warnf("Approving CSR %q from %q despite %s: %s", request.Name, request.Spec.Username, warner.Name, message)
+					requestsWarned.WithLabelValues(warner.Name).Inc()
```

Comment:
There can be multiple warnings per approved CSR, so we should change the metric help to "Number of warnings on approved requests".

Reply:
Done